### PR TITLE
TSK-754: PrettyPrint for JSON added

### PR DESCRIPTION
**Pull request by JulianSchallenmueller**

### CODE REVIEW

File: rest/taskana-rest-spring/src/main/java/pro/taskana/rest/RestConfiguration.java [hunk 1]
```diff
@@ -32,11 +39,14 @@
 @Configuration
 @ComponentScan
 @EnableTransactionManagement
-public class RestConfiguration {
+public class RestConfiguration extends WebMvcConfigurationSupport {
```
Comment (MiguelMartinRojas) [inline rest/taskana-rest-spring/src/main/java/pro/taskana/rest/RestConfiguration.java]: extending WebMvcConfigurationSupport  doesn't allow me to run taskana-rest-spring-example correctly

File: rest/taskana-rest-spring/src/main/java/pro/taskana/rest/RestConfiguration.java [hunk 2]
```diff
@@ -88,4 +98,10 @@ public Jackson2ObjectMapperBuilder jacksonBuilder(HandlerInstantiator handlerIns
     public HandlerInstantiator handlerInstantiator(ApplicationContext context) {
         return new SpringHandlerInstantiator(context.getAutowireCapableBeanFactory());
     }
+
+    @Override
+    protected void configureMessageConverters(List<HttpMessageConverter<?>> converters) {
```
Comment (MiguelMartinRojas) [inline rest/taskana-rest-spring/src/main/java/pro/taskana/rest/RestConfiguration.java]: I am not sure why are you using this. Where can I have a look to this when it is activated?

File: rest/taskana-rest-spring/src/main/java/pro/taskana/rest/RestConfiguration.java (hunk 1 above)
Comment (JulianSchallenmueller) [reply rest/taskana-rest-spring/src/main/java/pro/taskana/rest/RestConfiguration.java]: What problems are caused by it when you try to run it?

Comment (MiguelMartinRojas) [reply rest/taskana-rest-spring/src/main/java/pro/taskana/rest/RestConfiguration.java]: I usually run the taskana-spring-example running the main class of the project, I we extends webMvcConfigurationSupport The application doesn´t work when I try to open it in the browser.

**Pull request merged**